pkgs/by-name/go/goldboot/package.nix

```
@@ -0,0 +1,42 @@
+{
+  fetchFromGitHub,
+  rustPlatform,
+  pkgs,
+  lib,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "goldboot";
+  version = "0.0.10";
+
+  meta = with lib; {
+    mainProgram = "goldboot";
+    description = "Immutable infrastructure for the desktop!";
+    homepage = "https://github.com/fossable/goldboot";
+    license = licenses.agpl3Plus;
+    sourceProvenance = with sourceTypes; [ fromSource ];
+    platforms = platforms.unix;
+    maintainers = with maintainers; [ cilki ];
+  };
```

Can you also move `meta` block to the end?
```suggestion
  meta = {
    mainProgram = "goldboot";
    description = "Immutable infrastructure for the desktop!";
    homepage = "https://github.com/fossable/goldboot";
    license = lib.licenses.agpl3Plus;
    sourceProvenance = with lib.sourceTypes; [ fromSource ];
    platforms = lib.platforms.unix;
    maintainers = with lib.maintainers; [ cilki ];
  };
```
- #208242